### PR TITLE
Update elyra individual extension package descriptions

### LINKED ISSUE #1587: elyra vs elyra-server package
Both conda & pip have both elyra & elyra-server packages. On conda they seem independent, neither tries to install the other. What is the difference? How do they relate? Which should be installed when? Should both be installed?

### PR DESCRIPTION
**Pull request by akchinSTC**
Closes #1587

### What changes were proposed in this pull request?

Updates all of our lab extension package descriptions as well as the common core elyra-server package description during our release process to better describe what each package does and what is required. 
Unfortunately pypi doesnt look like it provides a way via the admin portal to alter existing package descriptions for prior releases. 
### How was this pull request tested?

Several test releases were built using the new release script and wheel package descriptions were checked and verified.
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by kiersten-stokes — APPROVED** [review]
Great idea! I have some NITs below for wording, but functionality looks good!

**Comment by ptitzler** [comment]
Should we add a link to each description to provide more context? Currently the only link on the package's homepage (e.g. https://pypi.org/project/elyra-server/) is to the Elyra GH repo:

image

For example, for the code snippets extension it might make sense to link to https://elyra.readthedocs.io/en/latest/user_guide/code-snippets.html ("latest" being replaced with the actual version identifier).

File: create-release.py [hunk 1]
```diff
@@ -442,11 +449,40 @@ def prepare_extensions_release() -> None:
...
+        ),
+        "elyra-code-viewer-extension": SimpleNamespace(
+            packages=["code-viewer-extension"],
+            description="The Code Viewer extension adds the ability to display a given chunk of code "
+            "(string) in a transient read-only 'editor' without needing to create a file."
+            "This extension will be available in JupyterLab core in a near future release and removed "
+            "from Elyra as a standalone extension.",
+        ),
+        "elyra-pipeline-editor-extension": SimpleNamespace(
+            packages=["pipeline-editor-extension", "metadata-extension", "theme-extension"],
```
Comment (ptitzler) [inline create-release.py:466]: This should require `elyra-code-viewer-extension`? afaik the editor is the only consumer of that functionality.

**Review by ptitzler — APPROVED** [review]
lgtm

**Pull request merged**